awx/ui_next/src/screens/Inventory/InventoryGroups/InventoryGroups.jsx

```diff
@@ -11,32 +11,18 @@ import InventoryGroupsList from './InventoryGroupsList';
 function InventoryGroups({ setBreadcrumb, inventory }) {
   return (
     <Switch>
-      <Route
-        key="add"
-        path="/inventories/inventory/:id/groups/add"
-        render={() => {
-          return (
-            <InventoryGroupAdd
-              setBreadcrumb={setBreadcrumb}
-              inventory={inventory}
-            />
-          );
-        }}
-      />
-      <Route
-        key="details"
-        path="/inventories/inventory/:id/groups/:groupId/"
-        render={() => (
-          <InventoryGroup inventory={inventory} setBreadcrumb={setBreadcrumb} />
-        )}
-      />
-      <Route
-        key="list"
-        path="/inventories/inventory/:id/groups"
-        render={() => {
-          return <InventoryGroupsList />;
-        }}
-      />
+      <Route key="add" path="/inventories/inventory/:id/groups/add">
```

Comment:
I'm pretty sure I updated these internal inventory ones to using children as opposed to render as part of my recently merged associate/disassociate host groups pr based on review feedback.  You might want to rebase.